ethereum/contracts/Messages.sol

```diff
@@ -6,10 +6,12 @@ pragma experimental ABIEncoderV2;
 
 import "./Getters.sol";
 import "./Structs.sol";
-import "./libraries/relayer/BytesParsing.sol";
+import "./libraries/external/BytesLib.sol";
+import "wormhole-solidity-sdk/libraries/BytesParsing.sol";
 
 
 contract Messages is Getters {
+    using BytesLib for bytes;
```

Comment:
Why?

Reply:
No idea, looks like it's left over from trying to get this to compile. Once https://github.com/wormhole-foundation/wormhole-solidity-sdk/pull/45 lands I'll give this PR a cleanup